### PR TITLE
various improvements

### PR DESCRIPTION
**Pull request by sonnyp**
Commits should be self explanatory.

I couldn't use GNOME Builder for some reason (unrelated to these changes but didn't look into it) and I wanted to make sure the sandbox was enabled so I did

```sh
cd gnomit
flatpak-builder --user --force-clean --repo=repo --install-deps-from=flathub flatpak ind.ie.Gnomit.json
flatpak --user remote-add --no-gpg-verify --if-not-exists Gnomit repo
flatpak --user install --reinstall --assumeyes Gnomit ind.ie.Gnomit
```

### CODE REVIEW

File: data/ind.ie.Gnomit.desktop.in [hunk 1]
```diff
@@ -6,3 +6,4 @@ Type=Application
 Categories=Development;Utility;RevisionControl;TextEditor;GNOME;GTK;
 StartupNotify=true
 Icon=ind.ie.Gnomit
+NoDisplay=true
```
Comment (TingPing) [inline data/ind.ie.Gnomit.desktop.in:9]: ?

Comment (sonnyp) [reply data/ind.ie.Gnomit.desktop.in:9]: see https://github.com/indie-mirror/gnomit/pull/6

**Comment by sonnyp** [comment]
@aral are you interested in contributions?

**Comment by TingPing** [comment]
Might have better luck submitting it to the correct repo: https://source.ind.ie/gnome/gnomit/gjs

but it doesn't seem too lively.

**Comment by sonnyp** [comment]
I would but the GitLab instance doesn't allow registering.

**Comment by aral** [comment]
Hey @sonnyp, thank you for the pull request. Sorry for the slow response; neck deep in other work at the moment. Will take a look when I get a moment.

**Comment by aral** [comment]
@sonnyp Finally managed to review. All looks good, thank you. Agree with hiding the desktop entry and the other changes. Appreciate your help :)

PS. Reason you couldn’t run in Builder is probably because of this hardcoded path:

```json
"x-run-args" : [
  "ind.ie/gnomit/gjs/tests/tag-message"
],
```

It was my hackish way of testing with different git messages. It’s a path from the home directory (haven’t found a way to specify a relative path in that json file).

**Pull request merged**